examples/seismic/tutorials/05_dask.ipynb

```diff
@@ -0,0 +1,496 @@
wUmDm3yOOERt+DdcG2tU6+tvcqig3VRPrTRAMxGUtKK5l6njqi\nYYm2ccVWGxFZg8o+4ecA/Hwp5VvZsp03UiJyICoLvhMAnG4s9vag6i1ZNqnthBBCWrFwY1IfBvDL\nAC4opXxhmoKdNlIisj+ATwA4FcC/njD3aTsmmymeBOCujM08IYSQlVicRkpE/jOASwD8m1JKahxK\n0+Vk3jUAPgrgLADnlFK+MiHbDQAuEpHTSyk31+U2ADgXwMeaHDfrHHW3Sh/qlLf78CSPrkPJe/uz\n+/KsjZo4i81OjM1gJ1Fr+cILGR9JFBkHovbaMlZikcVY1kqsrTzUxOlo5v5Ek9Kn3Zctn5UlszKr\nJ/F5MqDdX0bus9fpPe9ZedwjqvesZJqJwxWdyzyt+2onDkDlxAEAXikiOwHsLKXcLCK/BeA3Uc2H\nul1EfloV31lKuWOlY3TZk/oTABeimln8iDmZe2rZ7wZUHiauFZHLsDSZVwC8r8NzIYSQfZhV60ld\nb5avrv/fjGp+7Cvr5YvrP81fAPiNlQ7QZSM1Opl31n+a3wXwrtrrxDkArkJ1MQegarTOLKXc3eG5\nEEIImTGllNDYupRyRttjdDmZd3My325MblUbkfW95VnKZGWS/Z31lqwElLHUi/AkJC2TWCuiJvKF\nZ6EUySTRJNUMmYm1kTTjSXxNJpVGcrB3/Oias5N0vXuVleHaWiFqstZjnhxr68yT+Lx7YLc1sdRr\n66NPE30/msTk8srofUc+Av3709a1wGLAUB2EEDI4Fsdwoi1spAghZHAMp5FiPClCCCELy6B7Up4J\nZ1afzprxZuLTZDXyLs2ao315M+Otvp0xg83G1OkSe8zMPcjGk9LjevY40XjVpDyA/xzp49g686YV\nZO/BrLCjHJlnNxpv88aX7PhhZopA5t5Mg/eOIFjvjeFmTdD12FV2RGlUpkcOZqdi0I0UIYTsu7CR\nIoQQspCwJ7UwZM1BPQezFk/6yjpU9chKTVnpL+MM1EoRa518WUe4WS8VnkzieaKwZbx8kcn1Widf\nVjJpK1VpomNG8cLgbPOuJxsK3ktHEqN3/Cbm6E1iNkXvS2a6QCSve0wrr00q48Vci2JDeebpuq7t\n9Uy6J0OV+2g4QQghZGHpfU+KEEKIZTg9qd43Ut4MbQA4QqWz0oznTSKKueTJMVkPCU3i23iyS9a6\nr4mVmHdu1hmoPjctbaxx1kdk5UZPIvTkwojoXncZTyrCkzazElImn3UM7OWL3jHPE8RqxVLz3qMm\ncl+E9+xE1rBZ675MXUcWtO5zXbJP/GLT+0aKEELIBPoQDTEBx6QIIYQsLL3sSQmWuvbZ0NhaDoqs\nZrwfH94kTMDvbmdlo6ykNavOezY+VjTZ0iujpYyHVDqKS5QhGwesrXPUKK5RVkKyjn5HZK85Yy1p\n5T4t5el6f9TJY/fxKCZzsFn2Qr57k2+B+L3wmLZTEMnrTUKvZ4cLPCtVT7K125o8ExPPrUyxswWn\nl40UIYSQADZShBBCFpqBjEmxkSKEkKHBntT8GemwXmBCANit0p6Ga8t441U6nx1jWIfZEAUgzHgh\nyP6Q0tcZjTV542h23EbXj95fNOahx0288/YCG0ZlonE0Pb6y3llv723kfNbDG4+IvE9E3jkmlYnM\nmnX6QZW290Dn8+qgrePXqEwTM3FvykfkCHeWHYyMlxr7/fDM07NeXYZObxspQgghAZT7CCGELCSU\n+xaHSALS27zutpVJtGSQmtUd5IukN282eZPjZB2qevJl5D0iUx+Rc1NPdrIS0ENOPk9+tFKblqQ8\nGe9QU+a5zjZd/iBTxvOwoIm8mGizby23RWbamX1F0pAnJ1lT6PVO+gAnbZc96c5eW2bKhSdxAv67\nE0n/+p1vYk6ecb5rl5900tH70mQ6yiifIMjUY3rfSBFCCDEUDEbuo8cJQgghC0tve1KjnqwnYVm8\n1jhrQeflsfm88p7XAZtvP2d9dA4Zjxd2ua0llef5APDlOm1ZZs/Z83DgWedtMPm0XLdJpY9w0gBw\nmEpr6S+KPdal3KclzkhW9J7ryGmpPo5X3p6/PqYnmUbn6Vk+2uNkY3d5NInjlTl+9F57XiGi8PGZ\ntN1f1tpx0lBGsZko9xFCCFlIaDhBCCFkoRnImFQvGyk9JuhZnwHjPyS8mC1ZiSGS17xtjzt57HKT\ngUHvvKMQ3J4EE8UB8p7zyLmpN3EymlCp74knvXmSHjAu3WlZ78eSZZrIfeLcuGIuTtePlssiq7lM\nbKbomfSs+LSsquXGSfsYEU2I9uoqcjDrEclo3rk1sXLVRN9xT+6LJuZOe56W6P31yMZm6yu9bKQI\nIYQEUO4jhBCy0FDuI4QQspCwJ7WYZM1wm5iqR3jjUJHG7s2Aj3Ro77wzTj7tctZc2LuGSAd/wlnv\nmWLbc/PGnry0XdblPcexgB+wL2JsLNH5pZr9ARuN2+ixK11vTzjpJl4/7D3U41XaxN8bVwTGr0HX\nb2RCnzEhj8bYMuNDUVDNJp5kvH1H9T7LNmLF79QqNFIicgyA3wJwKoAXATgQwE+UUnaYfBsBXAng\n1XWebQDeXkq5NXMcTuYlhBDShBMB/BKAPQD+ZlIGEREANwI4G8BbAbwG1e+WrXUjtyKD6kkRQgip\nmf2Y1JdLKUcCgIhcAuDlE/KcB+A0AGeVUrbWebcBuBPA5QDettJBettIjXqyWkqw8sVmlf5RkE/j\n3dfIoaonX0TxfjwT40gC8rxreCbCWWegUXc645w0kjI9DwvW/Fmbimu5TstOkWm456lDn2dkcq23\nZe+BR9YjSWTGr6U8XYeec9VILvTM+K0s+wOVvl+ltSxqPU54nlQis/UoZpm334x8Gb2jmdhOWYkw\nMkHXy1lZ0TtmKw8cqyD3lWInW0zkPAD3jhqoutwDInIjgPORaKQo9xFCyBB5pu
```

Comment:
Optimised -> mixing british and american spellings

Reply:
fixed (except for the footer to be consistent with tutorial title)